src/utils/get-lybsin-podcasts.js

```diff
@@ -0,0 +1,47 @@
+// Fetches podcast info from https://mongodb.libsyn.com/rss
+
+const RSS_URL = `https://mongodb.libsyn.com/rss`;
+let prodcastList = [];
+
+export const parsePodcasts = podcastXML => {
```

Comment:
I think this logic would be great to write a unit test for! Any kind of parsing in general it's great to capture that logic in a test to represent behavior, what do you think?